optaplanner-persistence/optaplanner-persistence-jsonb/src/test/java/org/optaplanner/persistence/jsonb/api/score/buildin/hardmediumsoftbigdecimal/HardMediumSoftBigDecimalScoreJsonbAdapterTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.persistence.jsonb.api.score.buildin.hardmediumsoftbigdecimal;
+
+import java.math.BigDecimal;
+
+import javax.json.bind.annotation.JsonbTypeAdapter;
+
+import org.junit.Test;
+import org.optaplanner.core.api.score.buildin.hardmediumsoftbigdecimal.HardMediumSoftBigDecimalScore;
+import org.optaplanner.persistence.jsonb.api.score.AbstractScoreJsonbAdapterTest;
+
+public class HardMediumSoftBigDecimalScoreJsonbAdapterTest extends AbstractScoreJsonbAdapterTest {
+
+    @Test
+    public void serializeAndDeserialize() {
+        assertSerializeAndDeserialize(null, new TestHardMediumSoftBigDecimalScoreWrapper(null));
+
+        HardMediumSoftBigDecimalScore score = HardMediumSoftBigDecimalScore.of(new BigDecimal("1200.0021"),
+                                                                               new BigDecimal("-3.1415"),
+                                                                               new BigDecimal("34.4300"));
```

Comment:
Our friend, the code style. :-) We don't align on columns.
Anyway... leave it as it is, let's wait for Radek to bring his unifying PR.